eslint-configs/eslint-config-seekingalpha-node/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+  parser: 'babel-eslint',
+
+  extends: [
+    'plugin:node/recommended',
```

Comment:
you'll have to remove it or move to `rules/eslint-plugin-node/index.js` coz basically you are referencing plugin code before you connect it to eslint